### ISSUE TITLE
maximum recursion depth exceeded

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#348]
In GitLab by @ mtpay on Aug 6, 2018, 16:03

Hi,

I've tried to execute this command: 

```
autosubmit monitor a148 -group_by=chunk -expand="[ 20150530 [ 1 [ 1 ] ] ]"
```

and I had this error:

```
[CRITICAL] Unhandled exception on Autosubmit: maximum recursion depth exceeded while calling a Python object
Traceback (most recent call last):
  File "/shared/earth/software/autosubmit/3.11.0-foss-2015a-Python-2.7.9/lib/python2.7/site-packages/autosubmit-3.11.0-py2.7.egg/autosubmit/autosubmit.py", line 383, in parse_args
    args.expand_status, args.hide_groups, args.notransitive)
  File "/shared/earth/software/autosubmit/3.11.0-foss-2015a-Python-2.7.9/lib/python2.7/site-packages/autosubmit-3.11.0-py2.7.egg/autosubmit/autosubmit.py", line 974, in monitor
    job_grouping = JobGrouping(group_by, copy.deepcopy(jobs), job_list, expand_list=expand, expanded_status=status)
  File "/shared/earth/software/Python/2.7.9-foss-2015a/lib/python2.7/copy.py", line 163, in deepcopy
    y = copier(x, memo)
  File "/shared/earth/software/Python/2.7.9-foss-2015a/lib/python2.7/copy.py", line 230, in _deepcopy_list
    y.append(deepcopy(a, memo))
  File "/shared/earth/software/Python/2.7.9-foss-2015a/lib/python2.7/copy.py", line 190, in deepcopy
    y = _reconstruct(x, rv, 1, memo)
  File "/shared/earth/software/Python/2.7.9-foss-2015a/lib/python2.7/copy.py", line 334, in _reconstruct
    state = deepcopy(state, memo)
  File "/shared/earth/software/Python/2.7.9-foss-2015a/lib/python2.7/copy.py", line 163, in deepcopy
    y = copier(x, memo)
  File "/shared/earth/software/Python/2.7.9-foss-2015a/lib/python2.7/copy.py", line 257, in _deepcopy_dict
    y[deepcopy(key, memo)] = deepcopy(value, memo)
  File "/shared/earth/software/Python/2.7.9-foss-2015a/lib/python2.7/copy.py", line 190, in deepcopy
    y = _reconstruct(x, rv, 1, memo)
  File "/shared/earth/software/Python/2.7.9-foss-2015a/lib/python2.7/copy.py", line 328, in _reconstruct
    args = deepcopy(args, memo)
RuntimeError: maximum recursion depth exceeded while calling a Python object

```
FY: @dmanubens 

Any help will be appreciated.

Thank you!

MT

**Comment by kinow** [comment]
In GitLab by @ lbatista on Aug 6, 2018, 17:30

mentioned in commit 631ef79e78218be540d4b693a5c0bc04ec73fb97

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Dec 3, 2018, 16:40

I just checked this issue and see that this was resolved in 631ef79e78218be540d4b693a5c0bc04ec73fb97

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Dec 3, 2018, 16:40

closed
